docs/gotchas/script-order-of-operators.md

```diff
@@ -0,0 +1,55 @@
+# Ordering of operators in a script
+
+[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/Midnighter/nextflow-gotchas/blob/main/docs/gotchas/script-order-of-operators.md)
+
+## Problem
+
+In many cases, the order in which you define steps of a Nextflow script does not influence the order of execution. This is due to the nature of the process and channel dataflow paradigm used by Nextflow.
+
+In contrast, where a Nextflow _operator_ is placed within a script _does_ influence when it is executed.
+
+One such example is the `mix`ing of channels together prior passing it to a process. If a `.mix()` is specified _after_ the execution of a process that uses that channel, the contents of that particular `.mix()` will NOT be included in the execution of the process.
```

Comment:
You already use italics for emphasis so I would stick to that instead of all caps. Or make it bold.

```suggestion
One such example is the [`mix`ing](https://www.nextflow.io/docs/latest/operator.html#mix) of channels together prior passing it to a process. If a `.mix()` is specified _after_ the execution of a process that uses that channel, the contents of that particular `.mix()` will **not** be included in the execution of the process.
```